### PR TITLE
Update sticker sheet

### PR DESCRIPTION
**Pull request by bradnunnally**
Updated the sticker sheets to include the new header pattern and to correct a "design bug" that was present in the illustrator file where the body copy in the table component was the wrong font size. 

Ready for review @shawnbot and @juliaelman

### CODE REVIEW

**Review by juliasolorzano — COMMENTED** [review]
@bradnunnally nice work! 🎉 Here are a few questions and comments I have, broken out into each file to make it easier to follow:

**ALL**
- Please add the extended header with no mega menu.
- Please add the basic header with the mega menu.
- The extended header with the mega menu should have the top level item highlighted.
- Should we call out the show/hide functionality of the search box in the extended header in the file somehow?
- Please change the font size in the table component from 16px to 17px.

**Sketch, PDF**
- The search icon for the extended header is offset. See the screenshot below for reference: 
screen shot 2017-02-21 at 1 08 34 pm

**Omnigraffle**
- Should we replace the v.8 with v.1.0? Not sure how to get this naming convention in sync with our 1.0 release.

**Comment by bradnunnally** [comment]
Sketch file updated. Going to sync the branch then move on to the AI file.

**Comment by bradnunnally** [comment]
@juliaelman The files have all been updated based on your feedback.

**Comment by juliasolorzano** [comment]
Putting some more notes in here to keep track of all the things:

- Please change the font size in the table component from 16px to 17px in all of the files.
- The underline is off a bit in the mega menu selected tab indicator, see screenshot below.
- Remove `Omnigraffle` from PDF filename

screen_shot_2017-02-22_at_4_19_25_pm

**Comment by bradnunnally** [comment]
Alright! 🎉  The sticker sheets have been updated, file names have been updated, and the readme file has been updated. 

I'm 99% sure everything is good to start merging this PR. 

/cc @juliaelman @shawnbot

File: README.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
 
 This repository hosts editable, open source design files of the [U.S. Web Design Standards](https://playbook.cio.gov/designstandards) for government designers or anyone wishing to make use of the UI components or visual style guide in this site.
 
-For the alpha release, the following file types are available for your use:
+For the 1.0 release, the following file types are available for your use:
```
Comment (juliasolorzano) [inline README.md]: I think we can remove `For the 1.0 release,` from this sentence.

File: README.md [hunk 2]
```diff
@@ -2,14 +2,16 @@
 
 This repository hosts editable, open source design files of the [U.S. Web Design Standards](https://playbook.cio.gov/designstandards) for government designers or anyone wishing to make use of the UI components or visual style guide in this site.
```
Comment (juliasolorzano) [inline README.md]: Let's update the url in this paragraph to https://standards.usa.gov

**Review by juliasolorzano — CHANGES_REQUESTED** [review]
I think we are almost there!

- Are we going to have an EPS file too or just the EPS preview? When I pulled down the latest change set it only had the EPS preview file.

**Review by juliasolorzano — APPROVED** [review]

**Comment by juliasolorzano** [comment]
Done 🚀

**Pull request merged**